app/models/like.rb

```diff
@@ -0,0 +1,8 @@
+class Like < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  belongs_to :post, class_name: 'Post'
+
+  def update_post_likes_counter
+    post.update(likes_counter: post.likes.count)
+  end
```

Comment:
- [ ] Kindly apply the changes I pointed out in post file as regards the counter to this too.